documentation/puppet-api/v3/all_environment_modules.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema":     "http://json-schema.org/draft-04/schema#",
+  "title":       "All Environment Modules",
+  "description": "An array of information about all the modules in all Puppet code environments",
+  "type":        "array",
+  "properties": {
+    "$ref": "#/environment_modules"
+  },
+  "additionalProperties": false
+}
```

Comment:
@camlow325  - Ahhh ok, yeah in this case it probably makes sense do go ahead with what you listed at the bottom of your comment. I think it makes sense to just leave a note there as well as mention the ruby library to validate the schema. I'll go ahead and update the markdown file and delete the all environment json schema.